src/main/java/uk/gov/di/ipv/cri/common/library/util/ClientProviderFactory.java

```diff
@@ -87,28 +126,51 @@ public KmsClient getKMSClient() {
     public SqsClient getSqsClient() {
 
         if (null == sqsClient) {
-            sqsClient =
+            OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
+
+            AwsSdkTelemetry awsSdkTelemetry = AwsSdkTelemetry.create(openTelemetry);
```

Comment:
As above should these not be in the if so there not created if not needed

Reply:
They should have been, have now moved these into the if.